tests/main/reference/launch/outputIIDM.xml

```diff
@@ -3,95 +3,95 @@
   <substation id="_BUS___10_SS" name="BUS   10_SS" country="AF">
     <voltageLevel id="_BUS___10_VL" name="BUS   10_VL" nominalV="13.800000000000001" topologyKind="BUS_BREAKER">
       <busBreakerTopology>
-        <bus id="_BUS___10_TN" v="14.505302439884725" angle="-4.7826215712151097"/>
+        <bus id="_BUS___10_TN" v="14.503588002635899" angle="-4.7843936720204567"/>
       </busBreakerTopology>
       <load id="_LOAD__10_EC" name="LOAD  10" loadType="UNDEFINED" p0="9" q0="5.7999999999999998" bus="_BUS___10_TN" connectableBus="_BUS___10_TN" p="9" q="5.7999999999999998"/>
     </voltageLevel>
   </substation>
   <substation id="_BUS___11_SS" name="BUS   11_SS" country="AF">
     <voltageLevel id="_BUS___11_VL" name="BUS   11_VL" nominalV="13.800000000000001" topologyKind="BUS_BREAKER">
       <busBreakerTopology>
-        <bus id="_BUS___11_TN" v="14.588153821267763" angle="-4.4770505822354947"/>
+        <bus id="_BUS___11_TN" v="14.585310069725402" angle="-4.4777318975757963"/>
```

Comment:
Why do the values change ?

Reply:
We change dynawo which was corrected to take correctly into account the generators ref... so it is logical that the results values have changed